### ISSUE TITLE
Home assistant: add more data to the long term statistics

### ISSUE THREAD

**Issue by krusta4711** [zivillian/ism7mqtt#127]
Historical data in home assistant is purged after 10 days unless a sensor has defined the attribute `state_class`. At present ism7mqtt only treads temperatures and percentages as valuable for the home assistant long term statistics. Only for "°C" and "%" the `state_class` will be send.

I think nearly all values are important to have them long term to be able to compare certain situations. So I added a few more units in this PR: https://github.com/zivillian/ism7mqtt/pull/126

Other user might think different and do not want to have the data long term. So it should maybe even be configurable per unit.

THE CHANGE IS NOT TESTED YET as due to my other long running test which I do not want to stop at present (#115 ). So the PR cannot be merged. Would be good if someone could test whether it works or whether it at least do no harm. ;-)

**Comment by krusta4711** [comment]
One day after I changed the code, enabling the long term statistic in home assistant via _customize_ entry works suddenly.  8-P

So the code changes I did are not needed right now. Nevertheless for long term it might be the better way to be able to control LTS via ism7mqtt rather than _customize_. To use customize seems not to be a preferred way by the home assistant developers.

**Comment by drose28357** [comment]
I don't understand the steps (because being newbe) you did to gain LTS with "customize entry".
Did you change the definitions in develloper tools / states per each entity ?

I would like to have LTS for wolf_cha_0x3_270049_leistungsaufnahme_wp_ehz. The States View only shows
`unit_of_measurement: kW
friendly_name: Wolf CHA Leistungsaufnahme (WP + EHZ)`

There ist no 
`state_class: measurement`

**Comment by krusta4711** [comment]
Yes, exactly. The state class is not set by ism7mqtt for several units and therefore you need to add it via `customize:` section. E.g. adding to the customize.yaml:

```
sensor.wolf_wp_cha_0x3_270049_leistungsaufnahme_wp_ehz:
  state_class: "measurement"   
  unit_of_measurement: "kW"
```

I have all customization in an separated customize.yaml file. You have integrate it into the configuration.yaml with the `!include` command.

**Comment by drose28357** [comment]
If i check mein config...yaml files in developper tool i get an error:
`Konfigurationswarnungen
Integration error: customize - Integration 'customize' not found.`

here is my configuration.ymal section:
`...
scene: !include scenes.yaml
customize: !include customize.yaml
....`

and mein customize.yaml:
`# Sensoren der Wärmepumpe für Statistik-Anzeigen aktivieren (für ISM7MQTT Integration)
sensor.wolf_cha_0x3_270049_leistungsaufnahme_wp_ehz:
  state_class: "measurement"
  unit_of_measurement: "kW"
`

The customize.yaml file ist in /homeassistant/ just near the configurtion.yaml
Any ideas ?

**Comment by drose28357** [comment]
I found the solution. The configuration.yaml should contain this:
homeassistant: 
  customize: !include customize.yaml

I checked with devellopers / entities and now i have 

friendly_name: Wolf CHA Leistungsaufnahme (WP + EHZ)
unit_of_measurement: kW
state_class: measurement

**Comment by drose28357** [comment]
Thanks !
